source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -124,10 +126,10 @@ def _direct_path(
             )
 
             if (
-                crowfly_distance / float(direct_path_request['{mode}_speed'.format(mode=kraken_mode)])
+                crowfly_distance / float(direct_path_request['{mode}_speed'.format(mode=mode)])
                 > request['max_{mode}_direct_path_duration'.format(mode=mode)]
             ):
-                return None
+                return response_pb2.Response()
```

Comment:
Before it was **None**. 
We changed for an empty _Response_. Is it correct for this peculiar direct path case ?

Reply:
Following some changes I made, this code is tested now(this code was not tested). The response is tagged later here: https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/scenarios/new_default.py#L693. If we return `None`, the code will be broken